src/simulations/data_manager.py

```diff
@@ -138,7 +143,9 @@ def _passive_find_feasible_data(self):
         res = None
         new_res = False
         seed_data = res, batch
-
+        end = time.time()
+        duration_feasible_data = end - start
+        self.timer_feasible_data.append(duration_feasible_data)
```

Comment:
be careful you are timing the `_passive_find_feasible_data` method, which is the function which deals with passive homes only, rather than the `find_feasible_data` method. Is this on purpose?

Reply:
This was not on purpose so thank you! I've made the changes now